internal/controller/feature/features.go

```diff
@@ -83,6 +83,7 @@ const (
 
 	// Owner: N/A
 	// Alpha: v1.10
+	// Beta: v1.13
```

Comment:
question: Since this doesn't have an owner, does the fact that you're promoting it make you the de facto owner? 😇 

That's cheeky but does that make sense?